clio-client/src/main/scala/org/broadinstitute/clio/client/metadata/CramMover.scala

```diff
@@ -0,0 +1,288 @@
+    )
+
+    val dest = src.copy(
+      cramPath = movedCram,
+      craiPath = movedCram.map { cramUri =>
+        // DSDEGP-1715: We've settled on '.cram.crai' as the extension and
+        // want to fixup files with just '.crai' when possible.
+        URI.create(s"$cramUri${CramExtensions.CraiExtensionAddition}")
+      },
+      logPath = src.logPath.map(buildFilePath(_, destination)),
+      analysisFilesTxtPath = src.analysisFilesTxtPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.AnalysisFilesTxtExtension)
+        )
+      ),
+      preAdapterDetailMetricsPath = src.preAdapterDetailMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.PreAdapterDetailMetricsExtension)
+        )
+      ),
+      preAdapterSummaryMetricsPath = src.preAdapterSummaryMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.PreAdapterSummaryMetricsExtension)
+        )
+      ),
+      alignmentSummaryMetricsPath = src.alignmentSummaryMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.AlignmentSummaryMetricsExtension)
+        )
+      ),
+      bamValidationReportPath = src.bamValidationReportPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.BamValidationReportExtension)
+        )
+      ),
+      preBqsrDepthSmPath = src.preBqsrDepthSmPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.PreBqsrDepthSMExtension)
+        )
+      ),
+      preBqsrSelfSmPath = src.preBqsrSelfSmPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.PreBqsrSelfSMExtension)
+        )
+      ),
+      preBqsrLogPath = src.preBqsrLogPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.PreBqsrLogExtension)
+        )
+      ),
+      cramValidationReportPath = src.cramValidationReportPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.CramValidationReportExtension)
+        )
+      ),
+      crosscheckPath = src.crosscheckPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.CrossCheckExtension)
+        )
+      ),
+      duplicateMetricsPath = src.duplicateMetricsPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.DuplicateMetricsExtension)
+        )
+      ),
+      fingerprintPath = src.fingerprintPath.map(
+        buildFilePath(
+          _,
+          destination,
+          newBasename.map(_ + CramExtensions.FingerprintVcfExtension)
```

Comment:
it looks like this will overwrite `fingerprintVcfPath` since it has the same file extension

Reply:
The fields refer to the same file. Looks like we don't actually use `fingerprintVcfPath` in prod based on querying:
```
{
    "query": {
        "exists": { "field": "fingerprint_vcf_path" }
    }
}
```
And getting no results. We could take the chance to remove the field now.